Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM gradle:jdk-alpine
```

Comment:
I would use a Docker [multi-stage build](https://docs.docker.com/develop/develop-images/multistage-build/) here. In a nutshell: your Docker has two stages:

1. Building your application.
1. Running your application.

In the first stage, you'll need to have access to the SDK and such, which in your case is `gradle:jdk-alpine`. This is all good. The second stages though, only requires the JVM runtime. What you can do with Docker multi-stage builds, is to copy contents from one stage to another. Doing it this way has one major advantage: the resulting Docker image will be smaller, as it will not contain the SDK but only the runtime.

As an example of how to apply this, see the [C# analyzer](https://github.com/exercism/csharp-analyzer/blob/master/Dockerfile).